gateway-api/api/get-my-profile.js

```diff
@@ -0,0 +1,19 @@
+var api = require('express')();
+var superagent = require('superagent');
+var bodyParser = require('body-parser');
+
+module.exports = function(db) {
+api.get('/my-profile/:id', (req, res)=> {
```

Comment:
This should be a route, not a whole API. Look to the alive route for a base template, or the other gateway routes already implemented